autoload/vundle.vim

```diff
@@ -24,7 +24,7 @@ com! -nargs=0         PluginDocs
 \ call vundle#installer#helptags(g:bundles)
 
 " Aliases
-com! PluginUpdate PluginInstall!
+com! -nargs=* -complete=custom,vundle#scripts#complete PluginUpdate PluginInstall! <args>
```

Comment:
I thinks these completions don't make sense for the updater. We should only complete installed plugins, and this yields all known plugins,